ui/app/helpers/utils/token-util.js

```diff
@@ -200,3 +201,30 @@ export function getFormattedTokenFiatAmount (
   })
   return `${formatCurrency(currentTokenInFiat, currentCurrency)} ${currentCurrency.toUpperCase()}`
 }
+
+/**
+ * Given a transaction receipt for a contract transaction that involves a token transfer to a given account,
+ * returns the amount of tokens transferred.
+ *
+ * @param {object} [txReceipt] - The transaction receipt object, as can be found in any standard MetaMask txMeta object
+ * @param {string} tokenAddres - The hex address of the token being transferred
+ * @param {string} receiverAddress - The address of the account being transferred to
```

Comment:
It might help to be explicit about the expected case of the addresses being passed in:
```suggestion
 * @param {string} tokenAddres - The lower-case hex address of the token being transferred
 * @param {string} receiverAddress - The lower-case address of the account being transferred to
```